src/filesystem/impls/filer/lib/content.js

```diff
@@ -37,8 +38,13 @@ define(function (require, exports, module) {
             return id === "markdown";
         },
 
+        isArchive: function(ext) {
+            ext = FilerUtils.normalizeExtension(ext);
+            return ext === '.zip' || ext === '.tar'; 
+        },
+
         mimeFromExt: function(ext) {
-            ext = ext.toLowerCase();
+            ext = FilerUtils.normalizeExtension(ext, false);
```

Comment:
You can just do `ext = FilerUtils.normalizeExtension(ext)` and not bother passing `false`, since passing nothing will also produce a `false` type value in the function.  This is more common in JS.